remo/api.py

```diff
@@ -215,18 +208,44 @@ def get_annotations(self, annotation_set_id: int, annotation_format='json'):
             'api/v1/ui/annotation-sets/{}/export/?annotation-format={}'.format(annotation_set_id, annotation_format))
         return self.get(url).json()
     
-    def export_annotation_json_to_csv(self, annotation, output_file='output.csv', task=None):
-        # TODO: multiple class case
+    def export_annotation_to_csv(self, annotation_set_id: int, output_file, annotation_task):
```

Comment:
@melodiCyb @vmanilo let's not use type annotation, it's not common AFAIK and doesn't make the code readble